README.md

```diff
@@ -76,317 +110,607 @@ cartridge --help
+Description=Tarantool Cartridge app ${name}@%i
+After=network.target
+
+[Service]
+Type=simple
+ExecStartPre=${mkdir} -p ${workdir}.%i
+ExecStart=${bindir}/tarantool ${dir}/init.lua
+User=tarantool
+Group=tarantool
+
+Environment=TARANTOOL_WORKDIR=${workdir}.%i
+Environment=TARANTOOL_CFG=/etc/tarantool/conf.d/
+Environment=TARANTOOL_PID_FILE=/var/run/tarantool/${name}.%i.pid
+Environment=TARANTOOL_CONSOLE_SOCK=/var/run/tarantool/${name}.%i.control
+Environment=TARANTOOL_INSTANCE_NAME=%i
+```
+
+In this file, you can use the following environment variables:
+
+* `name` - the application name;
+* `workdir` - path to the work directory (by default, `/var/lib/tarantool/<name>`);
+* `dir` - path to the distribution directory (by default, `/usr/share/tarantool/<name>`);
+* `bindir` - path to the Tarantool binary (for Tarantool Enterprise, the default
+  is `/usr/share/tarantool/<name>`);
+* `mkdir` - full path to the `mkdir` binary in your system.
+
 ### Docker
 
-`cartridge pack docker ./myapp` will build a docker image.
+`cartridge pack docker ./myapp` builds a Docker image where you can start
+one instance of the application.
 
-Specific options:
+#### Build and runtime images
 
-* `--tag` - resulting image tag;
+In fact, two images are created: build image and runtime image.
 
-* `--from` - path to the base dockerfile for runtime image
-  (default to `Dockerfile.cartridge` in the project root);
+First, the build image is used to perform application build.
+The build stages here are exactly the same as for other distribution types:
 
-* `--build-from` - path to the base dockerfile for build image
-  (default to `Dockerfile.build.cartridge` in the project root);
+* [Stage 1. Cleaning up the application directory](#stage-1-cleaning-up-the-application-directory)
+* [Stage 2. Building the application](#stage-2-building-the-application)
+  (the build is always done [in Docker](#building-in-docker))
+* [Stage 3. Cleaning up the files before packaging](#stage-3-cleaning-up-the-files-before-packing)
 
-* `--sdk-local` - flag indicates that SDK from local machine should be
-  installed on the image;
+Second, the files are copied to the resulting (runtime) image, similarly
+to packing an application as an archive. This image is exactly the
+result of running `cartridge pack docker`).
 
-* `--sdk-path` - path to SDK to be installed on the image
-  (env `TARANTOOL_SDK_PATH`, has lower priority);
+Both images are based on `centos:8`.
 
-**Note**, that one and only one of `--sdk-local` and `--sdk-path` options should be specified
-for Tarantool Enterprise.
+All packages required for the default  `cartridge` application build
+(`git`, `gcc`, `make`, `cmake`, `unzip`) are installed on the build image.
 
-#### Image tag
+A proper version of Tarantool is provided on the runtime image:
 
-The image is tagged as follows:
+* For opensource, Tarantool of the same version as the one used for
+  local development is installed to the image, and source files are copied.
```

Comment:
Maybe remove `and source files are copied.`?

Reply:
ok